### ISSUE TITLE
Middleware layer in front of elasticsearch

### ISSUE THREAD

**Issue by ubergesundheit** [codeformuenster/muenster-info-hub#64]
(Please correct if wrong)

Right now we're exposing elasticsearch directly to the app and scrapers. Adding a middleware in front of elasticsearch would gain us the ability to:

- Check for duplicated events
- Validation for missing fields
- ...?

**Comment by JonesH** [comment]
Good idea imho. Any idea how to do this elegantly? Can this be done with [elasticsearch pipelines](https://www.elastic.co/guide/en/elasticsearch/reference/current/pipeline.html)?
